buildscripts/verify-build.sh

```diff
@@ -319,7 +319,7 @@ function __init__()
     mkdir -p "$MINIO_CONFIG_DIR"
     mkdir -p "$MINT_DATA_DIR"
 
-    if ! go get -u github.com/minio/mc; then
+    if ! go get github.com/minio/mc; then
```

Comment:
what i was thinking was since `mc` uses gomod already why not simply use `git clone https://github.com/minio/mc` and then `cd mc; go install -v`  ?